devel/lttng-modules/Makefile

```diff
@@ -8,37 +8,37 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lttng-modules
-PKG_VERSION:=2.10.8
+PKG_VERSION:=2.13.9
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://lttng.org/files/$(PKG_NAME)/
-PKG_HASH:=fe1d269bca723e8948af871c322c37d3900e647cdc5eb3efbe821e434beee44c
+PKG_HASH:=bf808b113544287cfe837a6382887fa66354ef5cc8216460cebbef3d27dc3581
 
 PKG_MAINTAINER:=
 PKG_LICENSE:=LGPL-2.1 GPL-2.0 MIT
 PKG_LICENSE_FILES:=LICENSE
 
+PKG_BUILD_PARALLEL:=1
+
 include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/package.mk
 
 define KernelPackage/lttng
   SUBMENU:=Other modules
   TITLE:=Linux Trace Toolkit: next generation (kernel modules)
   URL:=https://lttng.org/
-  DEPENDS:= @!TARGET_uml @KERNEL_FTRACE_SYSCALLS
+  DEPENDS:=@!TARGET_uml @KERNEL_FTRACE_SYSCALLS @KERNEL_KPROBES
   FILES:= \
-	$(PKG_BUILD_DIR)/lttng-*.$(LINUX_KMOD_SUFFIX) \
-	$(PKG_BUILD_DIR)/lib/lttng-*.$(LINUX_KMOD_SUFFIX) \
-	$(PKG_BUILD_DIR)/probes/lttng-*.$(LINUX_KMOD_SUFFIX)
+	$(PKG_BUILD_DIR)/src/lttng-*.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/src/lib/lttng-*.$(LINUX_KMOD_SUFFIX) \
+	$(PKG_BUILD_DIR)/src/probes/lttng-*.$(LINUX_KMOD_SUFFIX)
 endef
 
 define Build/Compile
-	$(MAKE) -C "$(LINUX_DIR)" \
-		ARCH="$(LINUX_KARCH)" \
-		CROSS_COMPILE="$(TARGET_CROSS)" \
+	+$(KERNEL_MAKE) $(PKG_JOBS) \
 		M="$(PKG_BUILD_DIR)" \
-		V="$(V)" \
+		CONFIG_LTTNG=m \
```

Comment:
Why do you add here CONFIG_LTTNG?

Reply:
This is an out-of-tree module, here is the right place to pass Kconfig to the Makefile.

https://github.com/lttng/lttng-modules/blob/da1f5a264fff33fc5a9518e519fb0084bf1074af/Makefile#L32

Reply:
I see. Thanks!